sklearn/cluster/k_means_.py

```diff
@@ -1262,6 +1264,15 @@ class MiniBatchKMeans(KMeans):
     Notes
     -----
     See http://www.eecs.tufts.edu/~dsculley/papers/fastkmeans.pdf
+
+    See also
+    --------
+
+    KMeans:
```

Comment:
I went through many other methods documentations and noticed that in all the cases there was no colon after the reference. Therefore, I removed colons in both `KMeans` and `MiniBatch` to make them consistent and to address the documentation requrements.